docs/macios/xcsync.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Xcode Sync
+description: Xcsync is designed to provide .NET developers with improved support for editing Apple specific files
+author: haritha-mohan, mcumming
+ms.author: harithamohan, mcumming
+ms.date: 08/05/2024
+monikerRange: ">=net-maui-9.0"
+---
+# Xcode Sync (Xcsync)
+
+Xcode Sync (Xcsync) is a tool that enables developers to leverage Xcode for managing Apple specific files with .NET projects. The tool generates a temporary Xcode project from a .NET project and synchronizes changes to the Xcode files back to the .NET project.
+
+Supported file types include:
+
+- Asset Catalog
+- Plist
+- Storyboard
+- Xib
+
+The tool has 2 commands: generate and sync. Use generate for tracking your changes from your .NET to Xcode and sync for tracking your changes from Xcode to .NET.
+
+## Synopsis
+
+### xcsync-generate
+
+```dotnetcli
+dotnet build /t:xcsync-generate /p:<XCSYNC_PROPERTY>=<Value>
+
+Parameters:
+  <XCSYNC_PROPERTY> ::= xcSyncProjectFile | xcSyncXcodeFolder | xcSyncTargetFrameworkMoniker | xcSyncVerbosity
+
+  xcSyncProjectFile ::=<PROJECT>
+  xcSyncXcodeFolder ::=<TARGET_XCODE_DIRECTORY>
+  xcSyncTargetFrameworkMoniker ::=<FRAMEWORK>
+  xcSyncVerbosity ::=<LEVEL>
+```
+
+### xcsync-sync
+
+```dotnetcli
+dotnet build /t:xcsync-sync /p:<XCSYNC_PROPERTY>=<Value>
+
+Parameters:
+  <XCSYNC_PROPERTY> ::= xcSyncProjectFile | xcSyncXcodeFolder | xcSyncTargetFrameworkMoniker | xcSyncVerbosity
+
+  xcSyncProjectFile ::=<PROJECT>
+  xcSyncXcodeFolder ::=<TARGET_XCODE_DIRECTORY>
+  xcSyncTargetFrameworkMoniker ::=<FRAMEWORK>
+  xcSyncVerbosity ::=<LEVEL>
+```
+
+### Arguments
+
+- **`/p:xcSyncProjectFile=<PROJECT>`**
+  The project file to build. This is **required**. Allowed values are MAUI project files or a project file that targets a supported platform (iOS, tvOS, macOS, MacCatalyst).
```

Comment:
```suggestion
  The project file to build. Supported project types are .NET MAUI projects or any .NET project that targets a supported platform (iOS, tvOS, macOS, MacCatalyst).
```

This argument is not required, it will be picked up from the context of the command

Reply:
thanks! forgot to update after our discussion